wallet/src/util/querySwingsetParams.ts

```diff
@@ -18,6 +18,16 @@ export const querySwingsetParams = async (
 ): Promise<QueryParamsResponse> => {
   const http = new HttpClient(endpoint);
   const trpc = await Tendermint34Client.create(http);
+  /**
+   * XXX
```

Comment:
oh is this what the other ones are too? consider filing a ticket in this repo to fix that and linking to it at the sites. second thought, probably not worth the overhead.